internal/core/metadata/v2/application/deviceservice.go

```diff
@@ -102,6 +104,11 @@ func PatchDeviceService(dto dtos.UpdateDeviceService, ctx context.Context, dic *
 func DeleteDeviceServiceById(id string, ctx context.Context, dic *di.Container) errors.EdgeX {
 	if id == "" {
 		return errors.NewCommonEdgeX(errors.KindContractInvalid, "id is empty", nil)
+	} else {
+		_, err := uuid.Parse(id)
+		if err != nil {
+			return errors.NewCommonEdgeX(errors.KindInvalidId, "failed to parse ID as an UUID", err)
+		}
```

Comment:
These lines doesn't need to be in `else` clause . `else` can be removed .

Reply:
Remove.